src/mon/MgrMap.h

```diff
@@ -179,6 +179,23 @@ class MgrMap
     return false;
   }
 
+  bool module_enabled(const std::string &module_name) const
```

Comment:
please be consistent where to put the `&`. i'd suggest put it right next to `std::string`.